### PR TITLE
fix: Safari fix from dom-to-image-improved

### PR DESCRIPTION
**Pull request by michaelbrewer**
Some Safari rendering fixes from `dom-to-image-improved`

Source: https://github.com/gyft/dom-to-image-improved/commits/master

### CODE REVIEW

File: src/dom-to-image-more.js [hunk 1]
```diff
@@ -263,6 +263,10 @@
             function cloneStyle() {
                 copyStyle(window.getComputedStyle(original), clone.style);
 
+                if ((util.isChrome() || util.isSafari() ) && clone.style.marker && ( clone.tagName === 'line' || clone.tagName === 'path')) {
```
Comment (IDisposable) [inline src/dom-to-image-more.js:266]: I really don't like doing coarse "what browser is this" checks... is it possible to just always do these adjustments (e.g. don't care about if it's Chrome or Safari)

File: src/dom-to-image-more.js [hunk 2]
```diff
@@ -283,6 +287,12 @@
                 function copyStyle(source, target) {
                     if (source.cssText) {
                         target.cssText = source.cssText;
+
+                        // Fix strange box-shadow in Safari
+                        if (util.isSafari()) {
+                            target.cssText = target.cssText.replace(/box-shadow(.*?);/, 'box-shadow: none!important;');
```
Comment (IDisposable) [inline src/dom-to-image-more.js:293]: This regex seems a bit dangerous, might eat a lot more than just the box-shadow setting.

File: src/dom-to-image-more.js (hunk 1 above)
Comment (michaelbrewer) [reply src/dom-to-image-more.js:266]: @IDisposable  - sure, i just wanted to sync up changes in a central version of `dom-to-image` it is unclear which one will or should be the official version.

**Comment by IDisposable** [comment]
Cannot use this in the current form

**Pull request closed without merging**